.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   test:
     name: Test
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
Should we be concerned that we aren't testing against newer versions of Ubuntu?

Reply:
[This](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on) doesn't really test whether the role can run on the particular Ubuntu version. Usually just specifies what image should be used to run the molecule test; We'd need to change the image [here](https://github.com/onaio/ansible-django/blob/master/molecule/default/molecule.yml#L8) if we'd like to test using something above 18.04....

_Main reason for this change is an issue with systemd running on the ubuntu-latest image_